cluster/environments/azure-multiple-clusters-waf-tm-apimgmt/api-management.tf

```diff
@@ -0,0 +1,398 @@
+# resource "azurerm_resource_group" "api_mgmt_rg" {
+#   name     = "${var.api_management_resource_group_name}"
+#   location = "${var.region}"
+# }
```

Comment:
Why are these commented out?  If not needed, they should be removed.

Reply:
@ReenuSaluja , take a look at the branch in your forked repo: https://github.com/ReenuSaluja/bedrock/tree/bedrockpull-sg/cluster/environments/azure-multiple-clusters-waf-tm-apimgmt

Reply:
Resolved and build test run successfully